Tests/Controllers/PeopleController.cs

```diff
@@ -50,6 +51,13 @@ public Person PostPerson(string id, Person person)
         {
             person.Identifier = id;
             return person;
+        }
+
+        [HttpPost]
+        [Route("relationships/people")]
+        public JObject PostParams([FromBody] JObject bodyParams)
```

Comment:
yeah, the way ActiveModelSerializers does it, is that it flattens the attributes and relationships, and appropriately renames polymorphic keys,

```
relationships: {
   admin: {
      data: {
        id: 1, type: 'users'
      } 
   }
}
```

becomes:
```
admin_id: 1, admin_type: 'User'
```